git/githistory/ref_updater.go

```diff
@@ -74,7 +98,7 @@ func (r *refUpdater) updateOneTag(tag *gitobj.Tag, toObj []byte) ([]byte, error)
 	return newTag, nil
 }
 
-func (r *refUpdater) updateOneRef(list *tasklog.ListTask, maxNameLen int, seen map[string]struct{}, ref *git.Ref) error {
+func (r *refUpdater) updateOneRef(list *tasklog.ListTask, maxNameLen int, seen map[string][]byte, ref *git.Ref, gitUpRefScript *string) error {
```

Comment:
In my proposed version where we pass an input pipe to this function, the function's signature would be something like:
```suggestion
func (r *refUpdater) updateOneRef(list *tasklog.ListTask, maxNameLen int, seen map[string][]byte, ref *git.Ref, cmdInput io.WriteCloser) error {
```